src/tools/metainfo/mdinfo.cpp

```diff
@@ -504,21 +504,30 @@ class MDVEHandlerClass : public IVEHandler
         // If this is one of our errors, then grab the error from the rc file.
         if (HRESULT_FACILITY(hrRpt) == FACILITY_URT)
         {
+#ifndef FEATURE_PAL
+            // We cannot cast BYTE* to va_list for marker in Unix.
```

Comment:
It would be more appropriate to exclude the whole MDVEHandler under `FEATURE_METADATA_VALIDATOR`. The counterpart functionality in coreclr.dll is under this ifdef (disabled in CoreCLR - for both Unix and Windows). IMHO, it is not important (low priority) to enable FEATURE_METADATA_VALIDATOR for CoreCLR ildasm.